README.md

```diff
@@ -71,7 +71,12 @@ require('yabs'):setup({
             -- WARNING: this api is experimental and subject to
             -- changes
             require('yabs'):run_task('build', {
-              on_exit = function(Job, exit_code) -- Job here is a plenary.job object
+              -- Job here is a plenary.job object that represents
+              -- the finished task, read more about it here:
+              -- https://github.com/nvim-lua/plenary.nvim/blob/master/lua/plenary/job.lua
+              on_exit = function(Job, exit_code)
+                -- You can omit extra arguments or skip
+                -- some of them using _ for the name
```

Comment:
Not a huge deal but I think

```lua
-- The parameters `Job` and `exit_code` are optional
-- leaving one or both of them out will not cause an error
```

is clearer. Obviously it doesn't have to be that exact wording, but I think including the word "optional" makes it clear what you're trying to say.